Formula/kontena-cli.rb

```diff
@@ -0,0 +1,154 @@
+class KontenaCli < Formula
+  desc "Command-line client for Kontena container & microservices platform"
+  homepage "https://kontena.io/"
+  url "https://github.com/kontena/kontena/archive/v1.1.1.tar.gz"
+  sha256 "e08e26a78e678eb0a1e5a2aa687829b9473a0c4744060b48e4a35050ebad0221"
+  head "https://github.com/kontena/kontena.git"
+
+  bottle :unneeded
+
+  option "with-ruby-autodiscovery", "Select a Ruby interpreter on every run instead of linking to the Ruby used during installation"
+  option "with-development", "Install development libraries"
+
+  depends_on :ruby => "2.1"
+
+  def install
+    # The depends_on :ruby is a bit flawed. It will validate that a ruby with version >= X exists
+    # but does not patch PATH to return that one
+    ruby_command = pick_a_ruby("2.1")
+    gem_command = gem_for_ruby(ruby_command)
+
+    # Make --version indicate it is a HEAD build
+    inreplace (buildpath/"cli/VERSION"), /(\d+\.\d+\.\d+).*/, "\\1-head" if build.head?
+
+    # Also add a "homebrew" build tag to "kontena --version" string
+    inreplace buildpath/"cli/lib/kontena/main_command.rb", "RUBY_PLATFORM +", "RUBY_PLATFORM + '+homebrew' +"
+
+    # If building from tar.gz sources, git ls-files obviously doesn't work
+    unless build.head?
+      inreplace buildpath/"cli/kontena-cli.gemspec", "`git ls-files -z`.split(\"\\x0\")", "Dir['LOGO', 'README.md', 'LICENSE.txt', 'VERSION','bin/*', 'lib/**/*']"
```

Comment:
Let's make lines <= 80 characters. You can use a Git checkout (url with `:revision`, `:tag`) instead of patching if upstream assumes this is being built from Git.